app/helpers/group-by.js

```diff
@@ -0,0 +1 @@
+export { default, groupBy } from 'ember-functional-helpers/helpers/group-by';
```

Comment:
Can we also export in https://github.com/DockYard/ember-functional-helpers/blob/remove-compose/addon/index.js please